docs/pipelines/run-cmd.md

```diff
@@ -14,4 +14,49 @@ code and tries to install it (will finish with error if not running from NeMo-Sk
 ns run_cmd --cluster=local cd /nemo_run/code/ && pip install -e .
 ```
 
-There are many more examples of how to use `run_cmd` throughout our documentation.
\ No newline at end of file
+There are many more examples of how to use `run_cmd` throughout our documentation.
+
+## LLM Server and Sandbox Server
+
+While we can run arbitrary commands with the default `run_cmd` script, we also provide the ability to
+run a LLM server with the `--model` argument and a few extra arguments for the server config. These arguments 
+are similar to the ones used for `start_server` script.
+
+This can be useful to run a server on a local machine or on a cluster with GPUs in a slurm job, while also being able to
+run arbirary code that uses LLM calls. 
+
+### Example
```

Comment:
would be nice to use this example (or something similar) in the gpu tests to make sure this new functionality is checked in ci. Can be done in another PR, but please create an issue to track this if not including here